lib/cli/interactive-setup/index.js

```diff
@@ -18,9 +19,14 @@ module.exports = async (context) => {
     const stepData = await step.isApplicable(context);
     if (stepData) {
       process.stdout.write('\n');
-      context.stepHistory = [];
+      context.stepHistory = new StepHistory();
       context.history.set(stepName, context.stepHistory);
       await step.run(context, stepData);
     }
   }
+  const commandUsage = Array.from(context.history.entries()).map(([step, stepHistory]) => ({
+    name: step,
+    history: stepHistory.toJSON(),
+  }));
+  return { commandUsage };
```

Comment:
I think we may return `commandUsage` directly, also if I see correctly it's an assumption in `scripts/serverless.js` (?)